docs/conf.py

```diff
@@ -192,6 +192,10 @@
     "mailto": None,
 }
 
+myst_enable_extensions = [
+    "attrs_inline",
+]
```

Comment:
Needed for the `.external` link modifier in `reference/client.md` to work, so that the relative link to the client-side docs is resolved as an external link instead of treated as a Sphinx document link.

See: 

- https://myst-parser.readthedocs.io/en/latest/syntax/cross-referencing.html#customising-external-url-resolution
- https://myst-parser.readthedocs.io/en/latest/syntax/optional.html#syntax-attributes-inline